src/Catalyst.Abstractions/Dfs/BitswapData.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System.Collections.Generic;
+using Catalyst.Abstractions.Dfs.CoreApi;
+using Lib.P2P;
+using MultiFormats;
+
+namespace Catalyst.Abstractions.Dfs
+{
+    /// <summary>
+    ///   The statistics for <see cref="IStatsApi.BitswapAsync"/>.
+    /// </summary>
+    public class BitswapData
+    {
+        /// <summary>
+        ///   TODO: Unknown.
+        /// </summary>
+        public int ProvideBufLen;
+
+        /// <summary>
+        ///   The content that is wanted.
+        /// </summary>
+        public IEnumerable<Cid> Wantlist;
+
+        /// <summary>
+        ///   The known peers.
+        /// </summary>
+        public IEnumerable<MultiHash> Peers;
+
+        /// <summary>
+        ///   The number of blocks sent by other peers.
+        /// </summary>
+        public ulong BlocksReceived;
+
+        /// <summary>
+        ///   The number of bytes sent by other peers.
+        /// </summary>
+        public ulong DataReceived;
+
+        /// <summary>
+        ///   The number of blocks sent to other peers.
+        /// </summary>
+        public ulong BlocksSent;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4612924)